s2i-play/applier/templates/build.yml

```diff
@@ -99,7 +99,7 @@ parameters:
   displayName: Play builder source branch/tag reference.
   name: PLAY_BUILDER_SOURCE_REPOSITORY_REF
   required: false
-  value: master
+  value: atomic
```

Comment:
Please change back to `master` as this PR will be merged into the master branch of this repo.